metals/src/main/scala/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -37,13 +41,23 @@ final class OnTypeFormattingProvider(
     }
   }
 
-  private def indent(toInput: String, pos: meta.Position): String = {
-    val beforePos = toInput.substring(0, pos.start)
-    val lastPipe = beforePos.lastIndexOf("|")
-    val lastNewline = beforePos.lastIndexOf("\n", lastPipe)
-    val indent = beforePos.substring(beforePos.lastIndexOf("\n")).length
-    val length = toInput.substring(lastNewline, lastPipe).length
-    space * (length - indent)
+  private def findChar(
```

Comment:
maybe `findLastBetween(source: String, pattern: Char, lowerBound: Int, upperBound: Int)` ? I was kind of expecting it to find first between `currentPos` and `stop`